lib/diplomat/health.rb

```diff
@@ -41,6 +41,7 @@ def service(s, options = {})
       custom_params << ['passing'] if options[:passing]
       custom_params << use_named_parameter('tag', options[:tag]) if options[:tag]
       custom_params << use_named_parameter('near', options[:near]) if options[:near]
+      custom_params << use_valueless_parameter('cached') if options[:cached]
```

Comment:
cached is a bit like stale, meaning available on almost all methods in catalog (I mean not only health).
I would prefer to implement it the same way as cached (at worst, on local agent, it would have no effect)

Reply:
updated.